kitten-app/src/stateHooks/useForm.ts

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import createCatServices from "../services/createCatServices";
+
+// useForm functional component
+export const useForm = (callback: any, UnsavedCatname: { name: string; }) => {
+    const [values, setValues] = useState(UnsavedCatname);
+
+    // onChange
+    const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setValues({ ...values, [event.target.name]: event.target.value });
+    };
+    // onSubmit
+    const onSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        createCatServices(values);
```

Comment:
[question] If we were to re-use this `useForm` in other places where we have a form, are we able to do so and why/why not? And how can we improve on this?